apps/web/public/zuplo-banner.js

```diff
@@ -0,0 +1,359 @@
+class ZuploBanner extends HTMLElement {
```

Comment:
Almost completely copy=pasted from mockbin. can make it an actual module later when we land on a better design